docs/user-flows.md

```diff
@@ -14,7 +14,7 @@ In these cases, you want Lighthouse on a _flow_, not just a page load.
 
 Lighthouse can now run in three modes: navigations, timespans, and snapshots. Each mode has its own unique use cases, benefits, and limitations. Later, you'll create a flow by combining these three core report types.
 
-* **Navigation mode** analyzes a single page load. Prior to v9.6.1, all Lighthouse runs were essentially in this mode.
+* **Navigation mode** analyzes a single page load. Prior to v9.6.0, all Lighthouse runs were essentially in this mode.
```

Comment:
Reword? I can't tell at a glance without employing multiple brain cells if this is exclusive or not :')

Reply:
how about

> * **Navigation mode** analyzes a single page load. It should feel familiar as all Lighthouse runs prior to v9.6.0 were essentially in this mode. Even in v9.6.0 and onwards, it remains the default.